blockchain/tendermint/tendermint.go

```diff
@@ -122,20 +126,23 @@ func Initialize(shardname string, index int, WorkingDir string) {
 	for i := 1; i < nValidators; i++ {
 		nodeDir := path.Join(WorkingDir, fmt.Sprintf("%s%d", defNodeName, i))
 		config.SetRoot(nodeDir)
-		nodeKey, err := p2p.LoadNodeKey(config.NodeKeyFile())
+		nodeKey, err := types.LoadNodeKey(config.NodeKeyFile())
 		if err != nil {
 			_ = os.RemoveAll(WorkingDir)
 			fmt.Printf("Can't load node key ID\n")
 			return
 		}
-		persistentPeers[i] = p2p.IDAddressString(nodeKey.ID(), fmt.Sprintf("%s:%d", IPs[i], networks.Networks[index].Port))
+		persistentPeers[i] = nodeKey.ID.AddressString(fmt.Sprintf("%s:%d", IPs[i], networks.Networks[index].Port))
 	}
 
 	// Overwrite default config.
 	for i := 0; i < nValidators; i++ {
 		nodeDir := path.Join(WorkingDir, fmt.Sprintf("%s%d", defNodeName, i))
 		config.SetRoot(nodeDir)
-		config.LogLevel = "main:info,state:info,statesync:info,*:error"
+		config.Mode = "validator"
+		config.LogFormat = "plain"
+		config.LogLevel = "info"
+		// config.LogLevel = "main:info,state:info,statesync:info,*:error"
```

Comment:
`v0.35.0-rc1` switched to `zerolog`, and zerolog only supports a global log level.